.pre-commit-config.yaml

```diff
@@ -53,6 +53,8 @@ repos:
     rev: v3.17.0
     hooks:
     -   id: pyupgrade
+        args:
+          - "--py39-plus"
         stages: [manual]
```

Comment:
Why is this manual, by the way?

Reply:
Here's the full discussion about it : https://github.com/pytest-dev/pytest/pull/12418, the gist of it is that ruff does what pyupgrade does and the two are not out of sync often enough to make launching pyupgrade everytime worth it.

Reply:
Oh, interesting. I'd suggest that this is a good reason to put a code comment in this place :)

Reply:
```suggestion
        # Manual because ruff does what pyupgrade does and the two are not out of sync 
        # often enough to make launching pyupgrade everytime worth it
        stages: [manual]
```